<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QWidget" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="drawGrid">
         <property name="text">
          <string>Draw Grid</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="xCellSizeLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>X cell size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="xCellSizeSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="value">
          <double>25.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="yCellSizeLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Y cell size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="yCellSizeSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="value">
          <double>25.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="xCellsNumLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>X cells number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="xCellsNumberSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="yCellsNumberLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Y cells number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="yCellsNumberSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="azimuthLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Azimuth:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="azimuthSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="xOrigLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>X origin:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="xOriginSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>9999999.990000000223517</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="yOriginLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Y origin:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="yOriginSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>9999999.990000000223517</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="inlineParX">
         <property name="text">
          <string>Inlines parallel X</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveButt">
         <property name="text">
          <string>Save grid</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="GraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>GraphicsView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>xCellSizeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>xCellSizeSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>yCellSizeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>yCellSizeSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>xCellsNumLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>xCellsNumberSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>yCellsNumberLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>yCellsNumberSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>azimuthSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>azimuthLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>xOrigLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>xOriginSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>yOriginLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>yOriginSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
